AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md

```diff
@@ -99,4 +76,21 @@ In order to run on the DevCloud, you need to request a compute node using node p
     Please find more information about the mkldnn log [here](https://oneapi-src.github.io/oneDNN/dev_guide_verbose.html).
 
 ## Example of Output
-With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.
+With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.  
+

```

Comment:
After line 67 "pytorch PyTorch_Hello_World.py", I get this error message:
Traceback (most recent call last):
  File "PyTorch_Hello_World.py", line 12, in <module>
    import torch
ImportError: No module named torch

Do you know what I am missing?

Reply:
Hi, probably "conda activate pytorch".
Other possible cause is no correct pytorch installation from AI kit.

Reply:
have your "source activate pytorch" first?
another possible cause is unsuccessful installation for pytorch from AI kit.